### ISSUE TITLE
_tmp folders not renamed when downloads "finished" 

### ISSUE THREAD

**Issue by audi4444player** [jobobby04/TachiyomiSY#1277]
### Steps to reproduce

Download manga
Finishes downloading
Folder is not renamed resulting in not functioning as downloaded, inconsistant

### Expected behavior

Renaming folders

### Actual behavior

Folder name is left untouched, downloaded images cannot be seen if internet disconnected rather than reading the folder, allows you to press the download button again. (I'm unsure if it's reading the folder or downloading ontop of what's there). unfortunately it does not always happen. It is possible it is only happening with ex, rather than any extensions, but still not consistently. 

### TachiyomiSY version

App version: 1.10.5 (standard, dea38912f, 69, 2024-09-01T17:23:56Z) Preview build: 564 Android version: 14 (SDK 34; build UP1A.231005.007.X710XXS4BXG5) Device brand: samsung Device manufacturer: samsung Device name: gts9wifi (gts9wifixx) Device model: SM-X710 WebView: Android System WebView 128.0.6613.127

### Android version

14

### Device

Samsung galaxy tab s9 one ui 6.1

### Other details

 it is similar but not the same as the situation that I believe is known that it still acts as if it's downloaded but just doesn't show it on the icon/library, these two issues are making knowing what's downloaded very difficult

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https:/mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[1.10.5](https://github.com/jobobby04/tachiyomisy/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by KarunaTG** [comment]
Backup data > Clear all data > restore app data

That's how I fixed this issue

**Comment by WaitWhat4355** [comment]
Unfortunately, I am having the same problem. It isn't localized to one source/extension, and it isn't even consistent in having the issue, but heavily leans towards it. After downloading the chapters, it lists it as downloaded, but still tries to read it from online instead. The files are downloaded, too.

Sadly, the above fix suggested by KarunaTG does not seem to work for me. I have tried BackingUp, clearing app data/storage both within the app and through Android's Settings, and restoring from the Backup. I have yet to try a full, complete,reinstall and setup, without a backup to test.

Only difference is my device, android version, and TSY version. Though this was a recent migration from an older phone to a new one, coming from Tachiyomi 1.14 i believe.

Android
 v15

Device
Sony Xperia 1 V

TachiyomiSY Version
App version: 1.11.0 (standard, ffc1e2d, 70, 2024-10-28T03:12:22Z)
Preview build: 0
Android version: 15 (SDK 35; build 67.2.A.2.41 release-keys)
Device brand: Sony
Device manufacturer: Sony
Device name: XQ-DQ72 (XQ-DQ72)
Device model: XQ-DQ72
WebView: Android System WebView 132.0.6834.163

**Comment by WaitWhat4355** [comment]
It seems a completely fresh install has the same result.

**Comment by WaitWhat4355** [comment]
[audi4444player](https://github.com/audi4444player), do you also have TachiyomiSY set to use an SD card for downloading and reading chapters? As this seems to be the cause of all these issues for me. Switching to internal storage solved all these problems in the app for me, due to some feature newer versions of Android use called "Scoped Storage." Unfortunately, using the internal storage isn't really an option for me, as it's mostly full of more important data, and I want to keep what empty space it has for such things.

**Comment by audi4444player** [comment]
Yes, I do, I have a 1tb sd card specifically for this and there's only about 70gb internal storage available so also a problem for me unfortunately, It's such a quick thing to fix if I get the sd card out put it in a pc and use bru to fix, android seems to just unnecessarily cripple the sd card to the point where even if I find an app that renames it cant do it. Perhaps a "solution" would be a toggle for situations like ours where it doesn't use the _tmp extension and just directly creates the folder, maybe it adds a file that does the completion check instead, so it's not deleting or changing anything, just adding.
